modules/client-configuration/pages/snippets/add_channels_cli.adoc

```diff
@@ -4,6 +4,5 @@
 ----
 spacewalk-common-channels \
 <base_channel_name> \
-<client_channel_name> \
-<updates_channel_name> \
-----
\ No newline at end of file
+<child_channel_name>
```

Comment:
What does this mean?

Reply:
Please share a render of this, I don't really understand how this change is going to work.

Reply:
> What does this mean?

I just realised that specifying "tools channel" or whatever didn't always apply, so i genericised it to "child channels". That way whatever names we use for the child channels in the table (tools, universe, security, etc etc) will apply correctly.

Reply:
`child_channel_name` seems to show you only need to add one channel, when in fact you need to add some.

Also, given the table above, that doesn't mention anything about "child channels" I wonder if this is clear enough.

As a minimum I'd change `<child_channel_name>` to `<child_channel_name1> <child_channel_name2> ... <child_channel_nameN>` or similar

Reply:
That's a fair comment. We do use the parent/child wording in concepts about channels, though, so I think (hope?) that wouldn't be too confusing. I like the idea of adding a couple, though, I'll do that now.